source/blog/2017-04-27-rspec-3-6-has-been-released.md

```diff
@@ -0,0 +1,94 @@
+---
+title: RSpec 3.6 has been released!
+author: Sam Phippen, TODO: other editors
+---
+
+RSpec 3.6 has just been released! Given our commitment to
+[semantic versioning](http://semver.org/), this should be a trivial
```

Comment:
This is copied word for word from the previous post. I'll make your change, as I think it's better. We should standardise this blog post into a template.